lib/puppet/reports/servicenow.rb

```diff
@@ -23,7 +24,7 @@ def process
 
     short_description_status = noop_pending ? 'pending changes' : status
     incident_data = {
-      short_description: "Puppet run report #{time} (status: #{short_description_status}) for node #{host}",
+      short_description: "Puppet run report (status: #{short_description_status}) for node #{host} #{format_report_timestamp(time, metrics)}",
```

Comment:
As I look at this now, I'm considering changing this from `format_report_timestamp(time, metrics)` to `format_report_timestamp(time, metrics['time']['total'])`, and the signature of the method from `format_report_timestamp(time, metrics)` to `format_report_timestamp(time, duration)`.

Reply:
Given this change, I think we could change short description to

> Puppet run report status: <status> for node <host> (time: <time)

(I think the `(time: <time>)` bit is useful just to explicitly distinguish the time. Otherwise it can take an (unnecessary) bit to figure out that's the report timestamp when it comes immediately after the node's host name).

Reply:
Having the end time is probably easier for search the logs.

Reply:
yeah that could make it clearer what timestamp is being used.

Reply:
I can put `time: <time>` on there, but I would like to keep the timestamp at the end, not the front. Is that what you mean?

Reply:
Yeah, am OK with keeping it at the end (so `time: <time>`). Also agree with Greg that `time` should probably be a bit more descriptive (so if it is end time, then we use `end_time` or something similar).

Reply:
Updated. One example:
`Puppet run report (status: failed) for node pure-hazel.delivery.puppetlabs.net (report time: 2020-08-11 20:39:33 Z)`